openstf/feature_engineering/general.py

```diff
@@ -76,3 +80,37 @@ def remove_extra_feature_columns(
         )
 
     return input_data.drop(extra_features, axis=1)
+
+
+def enforce_feature_order(df):
+    """Enforces correct order of features.
+
+    Alphabetically orders the feature columns. The load column remains the first column
+        and the Horizons column remains the last column. Everything in between is alphabetically sorted.
+
+    Args:
+        df (pd.DataFrame): Dataframe with features.
+
+    Returns:
+        pd.DataFrame with features with all columns in the correct order
+    """
+
+    # Extract first column name
+    first_col_name = df.columns.to_list()[0]  # Most of the time this is load
+
+    # Sort columns
+    columns = list(np.sort(df.columns.to_list()))
```

Comment:
I would suggest to use the same convention within a function. So you either use:

`first_col_name` and `col_names` OR `first_column` and `columns` OR (most descriptive) `first_column_name` and `column_names`.

Now it's a bit mixed up.

Reply:
made this more consistent